metrics/grafana/tikv_details.json

```diff
@@ -38082,50 +38082,1785 @@
       "type": "row"
     },
     {
-      "collapsed": true,
+      "collapsed": false,
```

Comment:
I just noticed that should this change be removed?

Reply:
Ah yes...